### DISCUSSION TITLE
Customised node export to docx

### DISCUSSION THREAD

**Discussion by howie12345** [ueberdosis/tiptap#5820] [General Questions]
I have a customised node (image container) which basically is an image with caption. But every time i export, this image container is not there.  Any ways that I can convert the customised node into docx file?

**Comment by nperez0111** [comment]
If by export, you mean Tiptap's beta export/import extensions. Then, yes this a known limitation that custom nodes would not be exported. We are looking into a much more full featured DOCX import/export but it has to be custom to Tiptap which takes time
